src/w_zip.c

```diff
@@ -0,0 +1,197 @@
+//
+// Copyright(C) 2024 Roman Fomin
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+#include "doomtype.h"
+#include "i_printf.h"
+#include "m_array.h"
+#include "m_misc.h"
+#include "m_swap.h"
+#include "w_wad.h"
+#include "w_internal.h"
+
+#include "miniz.h"
+
+static void AddWadInMem(mz_zip_archive *zip, const char *name, int index,
+                        size_t data_size)
+{
+    I_Printf(VB_INFO, " adding %s", name);
```

Comment:
Could we somehow make it clearer when WADs are added from a ZIP file? For example, if I call the game with `-file eternal.zip`, I'd be surprised to find `adding NULL.WAD` in the output.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1694/commits/fa4f00ff50a3fad1b0bc6a56665cf4dd5ffb240a